docs/samples/browser-plugin-meetings/app.js

```diff
@@ -1496,7 +1502,8 @@ function toggleSendAudio() {
     const newMuteValue = !localMedia.microphoneStream.muted;
 
     localMedia.microphoneStream.setMuted(newMuteValue);
-    handleAudioButton(newMuteValue);
+    console.log('pkesari_mute state in stream and new mute value: ', localMedia.microphoneStream.muted, newMuteValue);
+    handleAudioButton(localMedia.microphoneStream.muted);
```

Comment:
Question: Why do we need to change the argument from `newMuteValue` to `localMedia.microphoneStream.muted` here?

Reply:
When a user is muted from the mute button in the participant list, the button under the streams section doesn't reflect the correct state hence this change was required to keep that button text updated properly

Reply:
now every call to `handleAudioButton()` passes `localMedia.microphoneStream.muted` as the argument, so I think it would be better to change `handleAudioButton()` to not have any arguments and just hardcode `localMedia.microphoneStream.muted` inside it

Reply:
Makes sense....Updated it now